.eslintcache

```diff
@@ -0,0 +1 @@
});\n  return result;\n}\n\n/**\n * Return a d3 scale that returns the literal value that was given to it\n * @returns {function} literal scale.\n */\nexport function literalScale() {\n  function scale(d) {\n    return d;\n  }\n\n  function response() {\n    return scale;\n  }\n\n  scale.domain = response;\n  scale.range = response;\n  scale.unknown = response;\n  scale.copy = response;\n\n  return scale;\n}\n\nexport default {\n  extractScalePropsFromProps,\n  getAttributeScale,\n  getAttributeFunctor,\n  getAttr0Functor,\n  getAttributeValue,\n  getDomainByAttr,\n  getMissingScaleProps,\n  getScaleObjectFromProps,\n  getScalePropTypesByAttribute,\n  literalScale\n};\n"}},"/Users/amila/Documents/Projects/react-vis/src/lib/utils/series-utils.js":{"size":5108,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/lib/utils/series-utils.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/axis-utils.js":{"size":1557,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/axis-utils.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/components.js":{"size":4071,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/components.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/data-utils.js":{"size":2351,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/data-utils.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/index.js":{"size":1261,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/index.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/scales-utils.js":{"size":7765,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/scales-utils.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/series-utils.js":{"size":2539,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/series-utils.js","messages":[],"errorCount":0,"warningCount":0}},"/Users/amila/Documents/Projects/react-vis/src/test/xy-plot.js":{"size":3155,"mtime":1484010101000,"hashOfConfig":"a6496ee3d26e9b8aee8c4cf61c4e042a","results":{"filePath":"/Users/amila/Documents/Projects/react-vis/src/test/xy-plot.js","messages":[],"errorCount":0,"warningCount":0}}}
```

Comment:
This was a mistake. I forgot to clear my .git cache after running the linter. Will PR to remove this